components/raftstore/src/store/fsm/peer.rs

```diff
@@ -1267,29 +1275,34 @@ where
         );
 
         let msg_type = msg.get_message().get_msg_type();
-        let store_id = self.ctx.store_id();
 
-        if disk::disk_full_precheck(store_id) || self.ctx.is_disk_full {
-            let mut flag = false;
+        if disk::is_disk_almost_full(self.ctx.disk_status, self.ctx.store.get_id())
```

Comment:
is_disk_almost_full and is_disk_already_full can be removed. We can match `disk_status` directly.